src/Constants.js

```diff
@@ -117,5 +136,24 @@ export const ROOT_ROUTE = "/";
 export const PROFILE_LIST_ROUTE = "/profiles";
 export const DASHBOARD_ROUTE = "/dashboard";
 export const EDITOR_ROUTE = "/profile-builder";
-export const EDITOR_PROFILE_ROUTE = "/profile-builder/:id";
+export const EDITOR_PROFILE_ROUTE = "/profile-builder/:profile_id";
 export const ERROR_ROUTE = "*";
+
+//Draggble Tabs
+export const DraggableTabNode = ({ ...props }) => {
```

Comment:
add common components folder  and move it there

Reply:
Done!